web/src/app/loading/components/LoadingButton.js

```diff
@@ -24,11 +25,13 @@ export default class LoadingButton extends Component {
       noSubmit,
       onClick,
       style,
+      form,
```

Comment:
Instead of adding `form` as a new/specific prop, let's just allow all extras to go to the underlying element. That way this component, as a button, will just any/all props a button supports in addition to the ones it uses directly.

```js
const {
  // noSubmit, onClick, etc...
  ...rest
} = this.props

<Button
   // ... existing props
  {...rest}
>
```